Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mesosphere/mesos-modules-dev-phusion:0.25.0-rc1
+FROM spikecurtis/mesos-modules-dev-phusion:taskinfo-container
```

Comment:
Think we should merge this into the integration branch?

I guess we can change it back later once Mesos is fixed, and we'd have a way to ensure we don't have regressions.

@karya0 what do you think?